settings.gradle.kts

```diff
@@ -49,6 +49,7 @@ gitHooks {
     commitMsg { conventionalCommits() }
     preCommit {
         tasks("ktlintCheck")
+        tasks("checkScalafmt")
```

Comment:
How long does it take?

Reply:
3 seconds in my pc: https://scans.gradle.com/s/ze7zskbahomi6/timeline#ynxm2bmfq4cim

Reply:
okay then. I see it does not depend on `compileScala` which is too slow